tagsets/helpers.php

```diff
@@ -347,4 +347,18 @@ function or_array_delete_values($array,$values) {
     return $array;
 }
 
+function fix_utf8($mixed) {
```

Comment:
This looks overcomplicated, I think you can achieve the same using:
```php
array_walk_recursive($mixed, function(&$value) { $value = mb_convert_encoding($value, 'UTF-8', 'UTF-8'); });
```

Reply:
Thanks! Yes, this looks like it achieves the same with less code. I am tending towards keeping the old version because it already worked well for a while, but will look into making it more efficient in a next update.